packages/web-frontend/cypress/scripts/generateConfig/generateOverlayConfig.js

```diff
@@ -49,7 +49,10 @@ const parseUrl = url => {
 
   const [path, queryParams] = url.split('?');
   const searchParams = new URLSearchParams(queryParams);
-  const [project, orgUnit] = path.split('/').filter(x => x !== '');
+  const [project, orgUnit] = path
+    .split('/')
+    .filter(x => x !== '')
+    .map(decodeURIComponent);
```

Comment:
@BinduMadhaviTammireddy this is a code (and not config) related change. It fixes a minor bug that occurs if you use urls that include encode HTML characters, eg `/laos_schools/LA_Phouvong%20District/Schools?report=LESMIS_num_lower_secondary_classrooms&reportPeriod=1st_Jan_2021-31st_Dec_202` (`%20` is an encoded space)

It's a small change but let me know if you think another dev should have a look at this one!

Reply:
thank you, it was informative.